src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -583,9 +589,12 @@ class DocBaseViewer extends BaseViewer {
 
         // If range requests are disabled, request the gzip compressed version of the representation
         if (PDFJS.disableRange) {
+            const encodingType = ENCODING_TYPES.GZIP;
             url = appendQueryParams(url, {
-                [QUERY_PARAM_ENCODING]: ENCODING_TYPES.GZIP
+                [QUERY_PARAM_ENCODING]: encodingType
```

Comment:
Can `this.encoding` be assigned once above, depending on if `PDFJS.diableRange` is enabled, then used here? That way we only assign it one time and also avoid an explicit `undefined` assignment.

Reply:
Yes, that sounds reasonable